ObservatoryFramework/Interfaces.cs

```diff
@@ -35,6 +35,11 @@ public interface IObservatoryPlugin
         /// </summary>
         public string Version { get; }
 
+        /// <summary>
+        /// An object containing metadata about this plugin.
+        /// </summary>
+        public AboutInfo AboutInfo { get => null; }
+
```

Comment:
Rather than have the plugins do the lift of mapping the `AboutInfo` object to `Name` and `ShortName` I think I'd prefer a default implementation for `Name` which grabs it from the `AboutInfo`, and update the `ShortName` default to check `AboutInfo` then fall back to `Name`.

Reply:
Done.

That creates a fairly compelling migration story for plugin authors as well.